invenio_app_ils/ui/backoffice/src/pages/ItemDetails/ItemDetails.js

```diff
@@ -35,11 +39,13 @@ export default class ItemDetails extends Component {
     }
     if (fetchLoading) return <Loader active inline="centered" />;
     return (
-      <Grid centered>
-        <Grid.Row>
-          <ItemMetadata data={data} />
-        </Grid.Row>
-      </Grid>
+      <Container>
+        <ItemTitle data={data} />
+        <ItemMetadata data={data} />
+        <ItemHoldings />
```

Comment:
I would not introduce the name `Holding` and stick to what we decided until now to avoid confusions. For us, it is just Loans.

Reply:
Renamed to `ItemLoans`